content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -124,6 +124,16 @@ The Account Resource supports proprietary codes for:
 * Account.state codes are maintained in [Code Set 24451 Benefit Order Status](#code-set-24451-benefit-order-status) and [Code Set 4002640 Correspondence Status](#code-set-4002640-correspondence-status)
 * Account.identifier codes are maintained in [Code Set 28200 PFT Bill Alias Type](#code-set-28200-pft-bill-alias-type)
 
+<h4>ChargeItem</h4>
+
+The ChargeItem Resource supports proprietary codes for:
+
+* ChargeItem.code codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.reason codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.procedure codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.modifier codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
```

Comment:
Modifier and revenue code are not fields so we should probably use extension here.

Reply:
It looks like Account.state and Account.balance extensions are handled the same way, If we make Sujans suggested change, we should probably change those too for consistancy.

Reply:
Please see my comment below.

Reply:
+1 as resolved in below comment.